OrbitGl/TracepointTrack.cpp

```diff
@@ -142,12 +146,27 @@ std::string TracepointTrack::GetSampleTooltip(PickingId id) const {
   TracepointInfo tracepoint_info =
       GOrbitApp->GetCaptureData().GetTracepointInfo(tracepoint_info_key);
 
-  return absl::StrFormat(
-      "<b>Tracepoint event</b><br/>"
-      "<br/>"
-      "<b>Core:</b> %d<br/>"
-      "<b>Name:</b> %s [%s]<br/>",
-      tracepoint_event_info->cpu(), tracepoint_info.name(), tracepoint_info.category());
+  if (thread_id_ == TracepointEventBuffer::kAllTracepointsFakeTid) {
+    return absl::StrFormat(
+        "<b>%s : %s</b><br/>"
+        "<i>Tracepoint event</i><br/>"
+        "<br/>"
+        "<b>Core:</b> %d<br/>"
+        "<b>Process:</b> %d [%s]<br/>"
+        "<b>Thread:</b> %d [%s]<br/>",
```

Comment:
Ah, I now see what's off. I would do the opposite, that is:

`%s [%d], <process/thread_name>, <process/thread_id>`

This is more consistent with other parts of Orbit.

Reply:
Thank you!